recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "0.60" %}
+{% set version = "1.0.0" %}
 
 package:
     name: prompt_toolkit
     version: "{{ version }}"
 
 source:
-    fn: prompt_toolkit-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/prompt_toolkit/prompt_toolkit-{{ version }}.tar.gz
-    md5: 516826aff6a09536037b6d02a9a3451d
+    git_url: https://github.com/jonathanslenders/python-prompt-toolkit
+    git_rev: {{ version }}
```

Comment:
@melund can you use the tarball (`https://github.com/jonathanslenders/python-prompt-toolkit/archive/1.0.0.tar.gz`) and the sha256 (`35a7ae49d5f654ac3f8d1619572ff0ec37403c230399819cef962426b560a047`) here?